### DISCUSSION TITLE
PasswordInput rightSection has a width of 32px, but in TextInput its 36px?

### DISCUSSION THREAD

**Discussion by jordam1407** [mantinedev/mantine#3481] [Q&A]
Hey guys, first of all, thank you for this amazing work. I am loving working with Mantine!

I've noticed that the PasswordInput and TextInput have different width sizes, which causes the icons to not be aligned vertically between them. I have styled the rightSection attribute using the createStyle function so that all the inputs have the same width, but maybe this could be fixed it in the future.

Also, Is there any place in the documentation where I can see how to customize some components, such as the button in the PasswordInput? 

I'm new to using Mantine and, to be honest, it's my first time using styled components, so I apologize if I'm embarrassing myself!

![Capturar](https://user-images.githubusercontent.com/91645575/216497322-db04c408-3c38-404d-bcbd-c8598520ffd3.PNG)

**Comment by rtivital** [comment]
You can change right section width with `rightSectionWidth` prop
